site/docs/voice/webhooks/gather.mdx

```diff
@@ -34,7 +34,7 @@ The gather event is sent after a [`<Gather>`](/docs/voice/bxml/gather) verb is e
 | startTime         | Time the call was started, in ISO 8601 format. |
 | answerTime        | Time the call was answered, in ISO 8601 format. |
 | tag               | (optional) The `tag` specified earlier in the call. If no `tag` was specified or it was previously cleared, this field will not be present. |
-| digits            | (optional) The digits collected from user. Empty string value if a timeout occurred before any digits were pressed. |
+| digits            | (optional) The digits collected from the user. Empty string value if a timeout occurred before any digits were pressed. |
```

Comment:
```suggestion
| digits            | (optional) The digits, letters, and/or symbols entered by the user. The string is empty if a timeout occurred before any buttons were pressed. |
```

Reply:
Letters? Do we have an exhaustive list of items this might return? I would have only expected 0-9 # and *